Lib/fontTools/varLib/mutator.py

```diff
@@ -308,6 +316,13 @@ def instantiateVariableFont(varfont, location, inplace=False):
 				addidef = any(op.startswith("GETVARIATION") for op in instructions)
 				if addidef:
 					break
+		# set macOS rendering bit (bit6) if set_macos_rendering_bit parameter is set to True
+		if set_macos_overlap_rendering_bit:
+			for glyph_name in glyf.keys():
+				glyph = glyf[glyph_name]
+				# Only set for glyphs with contours
+				if glyph.numberOfContours > 0:
+					glyph.flags[0] |= 1 << 6
```

Comment:
There should be a constant for OVERLAP_SIMPLE that you could import from the glyf table module, I believe

Reply:
Is this what you are referring to?

https://github.com/fonttools/fonttools/blob/master/Lib/fontTools/ttLib/tables/_g_l_y_f.py#L263

Reply:
Yes

Reply:
https://github.com/fonttools/fonttools/pull/1518/commits/1da324a7b462ffdedf7425df6bc032733debc670